zilencer/management/commands/populate_db.py

```diff
@@ -162,6 +162,8 @@ def handle(self, **options):
             } # type: Dict[Text, Dict[Text, Any]]
 
             bulk_create_streams(zulip_realm, stream_dict)
+            zulip_realm.notifications_stream = get_stream("Rome", zulip_realm)
+            zulip_realm.save(update_fields=['notifications_stream'])
```

Comment:
I made this change based on [this](https://chat.zulip.org/#narrow/stream/issues/subject/.233708/near/157484) discussion. Backend tests fail due to test realm not having a `Realm.notifications_stream` set up.